packages/govern-console/src/containers/DAOSettings/DAOSettings.tsx

```diff
@@ -144,7 +143,7 @@ const DaoSettings: React.FC<DaoSettingFormProps> = ({ onClickBack }) => {
   }, [daoList]);
 
   const proposalInstance = React.useMemo(() => {
-    if (provider && account && daoDetails) {
+    if (provider && daoDetails && account.address && account.signer) {
```

Comment:
any specific reason why writing `account.address && account.signer` was needed instead of `account` directly ?

Reply:
account is an object and it is never undefined, if .address and .signer are not checked they may be undefined inside account object hence breaking the app

Reply:
It didn't break before when I had `account` only. and if that's the case, we can return `account:undefined` from AugmentedWallet.

Reply:
ok the check is moved AugmentedWallet